src/flexmock/_api.py

```diff
@@ -28,6 +28,24 @@
 RE_TYPE = type(re.compile(""))
 
 
+async def future_raise(anything: type[BaseException]) -> None:
```

Comment:
Using `type` directly requires Python 3.9.
```suggestion
async def future_raise(anything: Type[BaseException]) -> None:
```